src/components/menu.tsx

```diff
@@ -0,0 +1,46 @@
+import { MdAdd } from "react-icons/md";
+
+import { Button } from "@/components/button";
+import { Drawer } from "@/components/drawer";
+import { ManageHistoryIcon } from "@/components/icons/manage-history";
+
+export type MenuDrawer = {
+  open?: boolean;
+  setDateDrawerOpen: (value: boolean) => void;
+  setMenuDrawerOpen: (value: boolean) => void;
+  setSatellitesDrawerOpen: (value: boolean) => void;
+};
+
+export const MenuDrawer = ({
+  open,
+  setDateDrawerOpen,
+  setMenuDrawerOpen,
+  setSatellitesDrawerOpen,
+}: MenuDrawer) => {
+  return (
+    <Drawer open={open}>
+      <div className="flex flex-col justify-center gap-8 py-4 pb-32 ">
+        <Button
+          className="mx-auto w-3/5"
+          icon={<MdAdd />}
+          onClick={() => {
+            setSatellitesDrawerOpen(true);
+            setMenuDrawerOpen(false);
+          }}
+        >
+          Satelites
```

Comment:
I only now realized there is a small typo.
```suggestion
          Satellites
```